Podfile.lock

```diff
@@ -45,34 +45,6 @@ PODS:
   - Kanvas (1.4.4)
   - MediaEditor (1.2.2):
     - CropViewController (~> 2.5.3)
-  - MRProgress (0.8.3):
-    - MRProgress/ActivityIndicator (= 0.8.3)
-    - MRProgress/Blur (= 0.8.3)
-    - MRProgress/Circular (= 0.8.3)
-    - MRProgress/Icons (= 0.8.3)
-    - MRProgress/NavigationBarProgress (= 0.8.3)
-    - MRProgress/Overlay (= 0.8.3)
```

Comment:
Also great to be getting rid of all these other sub-dependencies that we were carrying around without actually using them.

Reply:
Marius used to be a CocoaPods contributor, so he probably used it for testing subspecs 👀